vue/sbc-common-components/src/components/NotificationPanel.vue

```diff
@@ -0,0 +1,175 @@
+<template>
+  <div
+    v-if="showNotifications"
+    v-on:clickout="emitClose()"
+  >
+    <v-overlay></v-overlay>
+    <v-navigation-drawer
+      right
+      app
+      :width="440"
+      style="top: 70px; max-height: calc(100% - 0);"
```

Comment:
How does this look when banners are displayed?

Could `top` be calculated based on the sbc-header height (in CSS maybe)?

Reply:
How to calculate it?

Reply:
It might be possible to have a CSS `calc()` add up all the pieces that make up the header that you are trying to fit below.

Or, programatically, you could query various element heights, add them up, and dynamically set your panel's top.

This might be too complicated. But I really would like to see some sort of "$sbc-header-height" variable (constant) that is declared at an appropriate place (some main SCSS file) and that components such as this one can just use. This means only a single place will need to be updated when the header height changes.